cscs-checks/system/openstack/s3_check.py

```diff
@@ -0,0 +1,132 @@
+import reframe as rfm
+import reframe.utility.sanity as sn
+
+
+class S3apiCheck(rfm.RunOnlyRegressionTest):
+    def __init__(self):
+        super().__init__()
```

Comment:
You don't need to call `super()` anymore here.

Reply:
should I remove the supper() call even if the current reframe default version in daint/dom doesn't support this?

Reply:
No, keep it for the moment.